.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Wait you still need to remove it from git :)

Reply:
🙀  Again!